client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -98,7 +98,11 @@ class MobileLayout extends Component<MobileLayoutProps, MobileLayoutState> {
             title={i18next.t('learn.editor-tabs.code')}
             {...editorTabPaneProps}
           >
-            {usesMultifileEditor && <EditorTabs />}
+            {usesMultifileEditor && (
+              <div className='monaco-editor-tabs'>
+                <EditorTabs />
+              </div>
+            )}
```

Comment:
with the new changes, the EditorTabs, don't use smart CSS to position the component, so I have added `.monaco-editor-tabs` as a parent, instead of creating smart CSS again